### PR TITLE
Adds escaping

### PR DESCRIPTION
**Pull request by pdewouters**

### CODE REVIEW

**Comment by justintadlock** [comment]
Please see contributing guidelines for PRs: https://github.com/justintadlock/members/blob/master/contributing.md#pull-requests

File: admin/class-cap-tabs.php [hunk 1]
```diff
@@ -221,7 +221,7 @@ public function tab_nav() { ?>
 			<?php $icon = preg_match( '/dashicons-/', $section->icon ) ? sprintf( 'dashicons %s', sanitize_html_class( $section->icon ) ) : esc_attr( $section->icon ); ?>
 
 			<li class="members-tab-title">
-				<a href="<?php echo esc_attr( "#members-tab-{$section->section}" ); ?>"><i class="<?php echo $icon; ?>"></i> <span class="label"><?php echo esc_html( $section->label ); ?></span></a>
+				<a href="<?php echo esc_attr( "#members-tab-{$section->section}" ); ?>"><i class="<?php echo esc_attr( $icon ); ?>"></i> <span class="label"><?php echo esc_html( $section->label ); ?></span></a>
```
Comment (justintadlock) [inline admin/class-cap-tabs.php]: `$icon` is escaped on line 221.

File: admin/class-settings.php [hunk 2]
```diff
@@ -306,7 +306,7 @@ private function filter_links() { ?>
 					$url = remove_query_arg( 'view', $url ); ?>
 
 				<li class="<?php echo sanitize_html_class( $view->name ); ?>">
-					<a href="<?php echo esc_url( $url ); ?>" <?php echo $class; ?>><?php echo esc_html( $view->label ); ?></a>
+					<a href="<?php echo esc_url( $url ); ?>" <?php echo esc_attr( $class ); ?>><?php echo esc_html( $view->label ); ?></a>
```
Comment (justintadlock) [inline admin/class-settings.php]: `$class` is hard-coded on line 300.

File: admin/class-user-edit.php [hunk 3]
```diff
@@ -150,7 +150,7 @@ public function role_update( $user_id, $old_user_data ) {
 			$old_roles = (array) $old_user_data->roles;
 
 			// Sanitize the posted roles.
-			$new_roles = array_map( 'members_sanitize_role', $_POST['members_user_roles'] );
+			$new_roles = array_map( 'members_sanitize_role', wp_unslash( $_POST['members_user_roles'] ) );
```
Comment (justintadlock) [inline admin/class-user-edit.php]: No need for `wp_unslash()`.  `members_sanitize_role()` removes slashes.

File: admin/views/class-view-addons.php [hunk 4]
```diff
@@ -113,7 +113,7 @@ public function addon_card( $addon ) { ?>
 
 				<div class="desc column-description">
 
-					<?php echo wpautop( wp_strip_all_tags( $addon->excerpt ) ); ?>
+					<?php echo wp_kses_post( wpautop( wp_strip_all_tags( $addon->excerpt ) ) ); ?>
```
Comment (justintadlock) [inline admin/views/class-view-addons.php]: I'm fairly certain that `wp_strip_all_tags()` is going to be enough here.  I'm open to hearing about why it's not though.

File: admin/views/class-view-addons.php [hunk 5]
```diff
@@ -142,8 +142,9 @@ public function addon_card( $addon ) { ?>
 
 						<div class="column-downloaded">
 							<?php printf(
+								/* translators: number of active installs */
 								esc_html__( '%s+ Active Installs', 'members' ),
-								number_format_i18n( absint( $addon->install_count ) )
+								esc_html( number_format_i18n( absint( $addon->install_count ) ) )
```
Comment (justintadlock) [inline admin/views/class-view-addons.php]: `absint()` should be fine here.  I don't see the need for an `esc_html()` call.

**Comment by pdewouters** [comment]
Thanks for reviewing, I'll close it as it seems everything is already escaped.

**Pull request closed without merging**